packages/angular_devkit/schematics_cli/bin/schematics.ts

```diff
@@ -109,6 +152,7 @@ export async function main({
   const dryRun: boolean = argv['dry-run'] === null ? debug : argv['dry-run'];
   const force = argv['force'];
   const allowPrivate = argv['allow-private'];
+  const interactive: boolean = argv['interactive'] !== null || argv['interactive'];
```

Comment:
Would it better to add the `interactive` in https://github.com/angular/angular-cli/blob/d8764e103de8292845a4acc366d9e036aaa65484/packages/angular_devkit/schematics_cli/bin/schematics.ts#L336-L363

Add a a default to `true` and the usage here would be ` const interactive: boolean = argv['interactive'];

Reply:
Done. Thx for the feedback.